kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/internal/coordinatedcommits/CommitCoordinatorProviderSuite.scala

```diff
@@ -159,7 +159,7 @@ protected trait TestCommitCoordinatorClientBase extends CommitCoordinatorClient
     version: Long,
     lastKnownBackfilledVersion: lang.Long): Unit = {}
 
-  override def semanticEquals(other: CommitCoordinatorClient): lang.Boolean = this == other
+  override def semanticEquals(other: CommitCoordinatorClient): Boolean = this == other
```

Comment:
[nit] JBoolean

Reply:
This PR is already very big. Maybe we can avoid using java classes directly and rename them to JBoolean, JMap, JLogStore etc in a followup PR.